README.md

```diff
@@ -90,8 +90,14 @@ These are symmetric probability distributions centered at the observed reflectio
 `careless` includes normally-distributed and robust, t-distributed likelihoods. 
 
 ### Scaling Models
-Right now the only model which `careless` explicitly implements is a sequential neural network model. 
-This model takes reflection metadata as input and outputs a gaussian distribution of likely scale values for each reflection.
+
+### Scaling Models
+`careless` implements two primary approaches to scaling:
+
+1.  **Neural Network Scaling (Default)**: A sequential neural network that takes reflection metadata as input and learns a probability distribution of scale values. This is flexible and data-driven.
+2.  **Physics-Based Spectral Scaling**: For Laue data where the incident spectrum is known, you can use a tabulated spectral curve. This model interpolates the incident flux from a user-provided file and can optionally apply a geometric Lorentz correction ($\lambda^4$). This approach reduces the number of free parameters and is ideal for harmonic deconvolution when data redundancy is low.
```

Comment:
Great summary -- I think you should add that the spectral scaling is intended for neutron data. I would caution users against using it for X-ray, because it does not account for polarization. Open to discussing this further, and perhaps I am missing something.